spec/jobs/push_priority_appeals_to_judges_job_spec.rb

```diff
@@ -221,10 +221,10 @@ def to_judge_hash(arr)
       end
     end
 
-    let(:priority_count) { Appeal.count + VACOLS::Case.count }
+    let(:priority_count) { Appeal.count { |a| a.aod? || a.cavc? } + VACOLS::Case.count }
     let(:priority_target) { (priority_count + judge_distributions_this_month.sum) / judges.count }
 
-    it "should distibute ready priortiy appeals to the judges" do
+    it "should distibute ready priority appeals to the judges" do
```

Comment:
Found another one! 😄 

```suggestion
    it "should distribute ready priority appeals to the judges" do
```